### PR TITLE
Closes #61: moved package ext guidance to admiraldev and linked to it

### LINKED ISSUE #61: Remove package extension guidance from README (and link to admiraldev page instead)
### Background Information

Move package extension guidance section of the README to its own page in admiral dev so that all documentation is in one place. Add link to it in the README instead.

### Definition of Done

_No response_

### PR DESCRIPTION
**Pull request by manciniedoardo**
Closes #61 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by manciniedoardo** [comment]
PR to be merged only once [corresponding admiraldev PR](https://github.com/pharmaverse/admiraldev/pull/300) is live (link in README will be broken until then) and also [version update PR](https://github.com/pharmaverse/admiraltemplate/pull/64) too.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# admiraltemplate 0.0.3
+
+# Updates to site
+
+ - Package Extension guidance on in the front page was updated and moved to [this location](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html) in `{admiraldev}`, and a link was added to point to the new location (#61).
```
Comment (rossfarrugia) [inline NEWS.md]: we usually use the devel version of the site as default for admiraldev - so use this in the link. as it was important we always see the very latest developer guidance

File: README.Rmd [hunk 2]
```diff
@@ -83,33 +83,7 @@ The repository template includes the following:
...
-8. Once you are happy your package extension has been well tested and is at a sufficient state then make a submission to CRAN. The technical lead should be named as maintainer. After the CRAN release, you should advertise this via Slack & LinkedIn.
-9. Plan any future further enhancements and make issues. When your team feels ready you can open up to development contributions for these from the wider community - see `r dyn_link("this page.", admiral_homepage, "CONTRIBUTING.html#type-1-contribution-with-code")`. Please use the _“good first issue”_ (ideal for new starters) & _“help wanted”_ (ideal for more experienced contributors) issue labels.
+For detailed information regarding the ethos of `{admiral}` package extensions and how to decide if you should set your own one up, please see our [Package Extension Guidance](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html). You will also find details about team setup, suggested workflow and lessons learned from past package extensions.
```
Comment (rossfarrugia) [inline README.Rmd:86]: see comment above

**Review by rossfarrugia — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 2 above)
Comment (bms63) [reply README.Rmd:86]: With the adoption of GitHub Flow I believe we can drop the `main` `devel` stuff and just link to the generic site.

Comment (rossfarrugia) [reply README.Rmd:86]: yes makes sense - more important for users to be able to switch to different release versions on the site nowadays than devel vs main

**Comment by bms63** [comment]
I'm unclear what we need to update here with github flow and website updates.  Perhaps @manciniedoardo when you return we could get this one completed.

**Comment by manciniedoardo** [comment]
@bms63 I believe this should be good to go now (not sure what the checks are failing for though) as this PR was just to remove the package ext guidance from the readme of this repo and move it to a page on admiraldev.

**Comment by bms63** [comment]
> @bms63 I believe this should be good to go now (not sure what the checks are failing for though) as this PR was just to remove the package ext guidance from the readme of this repo and move it to a page on admiraldev.

news file wasn't formatted properly.  I updated and good to
